### ISSUE TITLE
Problem with new Crop Overscan

### ISSUE THREAD

**Issue by hacls1** [libretro/beetle-psx-libretro#820]
Hi guys, there is a problem with the new Crop Overscan, before, which was more accurate, you could just chop ~8px top and bottom, that's by default the configuration of DuckStation, for example, and there you have it.

Now you pretty much crop all overscan independent of how much it is, FFIX is 8px, MGS is another, Chrono Cross another, etc.. Which is great, the problem is there's a ton of games that still display trash pixels, like Dino Crisis 2, Legend of Dragoon, etc and automatically cuting off the horizontal blackness (vertical overscan) leaves you having to config additional cropping game by game, which is a nightmare.

I suggest leaving the original option there and put some "PAL optimize" option or something like that for someone who needs it, like the "Crop Mode>All Borders" from DuckStation.

![Exemplo1](https://user-images.githubusercontent.com/91802898/135731407-1a475326-40a2-44c5-89f3-748ffad3db3c.png)
![Exemplo2](https://user-images.githubusercontent.com/91802898/135731410-42e4fd91-27d9-4f4b-92e1-912c22225ee5.png)

**Comment by hacls1** [comment]
Hi, so the problem persists, I'll try to show what's happening...

This is Dino Crisis 2 - Config: Force 4:3 - Crop Overscan: On - Initial Scanline NTSC: 8 - Last Scanline NTSC: 231 = Perfect
![dc2-f43co231-8](https://user-images.githubusercontent.com/91802898/135928934-d6294885-4bed-4e7e-828e-bfe8f6d25a52.png)

But then FF9 with the same config as above = Chopping the image top and bottom (in this frame you can see only the top obviously)
![ff9-f43co231-8](https://user-images.githubusercontent.com/91802898/135929593-9db81c22-7630-4234-ae59-dbd79962687d.png)

------------------------------------

So I remove Crop Overscan.. The 4:3 ratio is gone and the pillarbox comes to the scene in DC2...
![dc2-f43nco231-8](https://user-images.githubusercontent.com/91802898/135929919-bf9654f3-0c93-4631-962e-2193d8b6f5c2.png)

In FF9 the top and bottom mess is corrected, but there's 4:3 no more and the pillarbox comes..
![ff9-f43nco231-8](https://user-images.githubusercontent.com/91802898/135930195-59e71425-2a11-4c17-81d7-415eb7328826.png)

--------------------------------------

Let's turn on the Crop Overscan again, but this time no chop in the scanline, Initial and Last Scanline back to standard 0 and 239 respectively.. DC2 back to 4:3, but now showing garbage pixels top and bottom..
![dc2-f43co239-0](https://user-images.githubusercontent.com/91802898/135930814-6286917d-c514-40c4-9dd4-d5ca080c04d6.png)

FF9 is perfect again because its naturally letterbox...
![ff9-f43co239-0](https://user-images.githubusercontent.com/91802898/135931080-e1251c5c-872f-42a7-9763-c187bccae6c1.png)

----------------------------------------

And if you remove Crop Overscan option and let the standard Scanlines numbers...

DC2 is 4:3 no more and shows garbage textures... FF9 is perfect but with overscan all around...
![dc2-f43nco239-0](https://user-images.githubusercontent.com/91802898/135931798-d7593cf9-bd2b-4134-b757-dd53ad97848a.png)
![ff9-f43nco239-0](https://user-images.githubusercontent.com/91802898/135931812-d5267481-1ae4-401a-b2be-97f5c5b931d0.png)

-----------------------------------------

In short, before, the Crop Overscan only removed the horizontal overscan leaving the vertical overscan up to you, then you just removed about 8px top and bottom and it's done. Now the Crop Overscan basically removes the pillarbox and letterbox and then what do you do with the trash pixels still being display? You'll have to config game by game.

**Comment by Amisaddai** [comment]
I'm also having issues with the crop overscan option after updating to 0.9.44.1
When the option is enabled, I get vertical scaling issues on certain video modes (displayed below with the tvout+interlacing shader), turning the option off seems to solve it, but then horizontal scaling gets messed up across all video modes.

![1](https://user-images.githubusercontent.com/78184818/139574257-423e054e-ab7b-4247-9226-d6de603b0693.jpg)
- _Integer scaling is enabled in all instances_

**Comment by daFourch** [comment]
You should push this issue here:

https://forums.libretro.com/t/problem-with-new-crop-overscan-in-beetle-psx/35283/3

**Comment by ds22x** [comment]
I guess I could try and cook up an additional option for people who want the old cropping behavior, while keeping the new cropping method intact.

**Comment by ds22x** [comment]
This can be marked as solved.
